tests/test_classification.py

```diff
@@ -10,15 +10,15 @@ def test_score(classification_truth_file_path, classification_prediction_file_pa
 
 
 @pytest.mark.parametrize(
-    'validation_metric',
+    'target_metric',
     [ValidationMetric.AUC, ValidationMetric.BALANCED_ACCURACY, ValidationMetric.AVERAGE_PRECISION],
 )
-def test_score_validation_metric(
-    classification_truth_file_path, classification_prediction_file_path, validation_metric
+def test_score_target_metric(
```

Comment:
```suggestion
def test_score(
```

Since `target_metric` can be mandatory, this test case is semantically more general.